ideascube/serveradmin/management/commands/csv2package.py

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
tempted to name the command `csv2pkg` to gain some characters to type, and `pkg` is still readable imho :)